From 98074af71f6e7a5bbaf05ffc2669c11c94e9ee4c Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Tue, 24 Jul 2012 18:07:09 -0700
Subject: [PATCH 108/795] MR1: CDH-5800. Add reactorRepo to MR1 Ivy resolvers.

With -DreactorRepo=$HOME/.m2/repository added to the ant call, the
local Maven repo will be checked first.
(cherry picked from commit 0d5af61d34788efbbc9c90f55622316fe36749ea)

(cherry picked from commit 9bc7636e9c72b4d2532de7b7eea401b5631abbd9)
---
 ivy/ivysettings.xml |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/ivy/ivysettings.xml b/ivy/ivysettings.xml
index 2401033..e8c2fa0 100644
--- a/ivy/ivysettings.xml
+++ b/ivy/ivysettings.xml
@@ -86,12 +86,14 @@
     <chain name="default" dual="true"
            checkmodified="true"
            changingPattern=".*SNAPSHOT">
+      <resolver ref="reactor-repo"/>
       <resolver ref="local"/>
       <resolver ref="maven2"/>
       <resolver ref="cdh-releases"/>
       <resolver ref="cloudera-snapshot"/> 
     </chain>
     <chain name="internal">
+      <resolver ref="reactor-repo"/>
       <resolver ref="local"/>
     </chain>
     <chain name="external">
@@ -99,6 +101,7 @@
       <resolver ref="cdh-releases"/>
     </chain>
     <chain name="external-and-snapshots">
+      <resolver ref="reactor-repo"/>
       <resolver ref="maven2"/>
       <resolver ref="cdh-releases"/>
       <resolver ref="apache-snapshot"/>
-- 
1.7.0.4

